src/graph.js

```diff
@@ -72,17 +76,31 @@ export default class Graph {
     };
   }
 
+  _getGroupByHourFunc() {
+    const now = new Date();
+    now.setHours(now.getHours() + 1);
+    now.setMinutes(0, 0, 0);
+    return (res, item) => {
+      const age = now - new Date(item.last_changed).getTime();
+      const interval = (age / (1000 * 3600)) - this.hours;
```

Comment:
nitpick: I saw it somewhere else too - it would be good to have a const for it. It will improve code readability too (talking about `1000 * 3600`)

Reply:
Sure!